src/App.vue

```diff
@@ -21,6 +24,9 @@ import 'reveal.js/dist/theme/white.css'
 
 const dataSeparator = '\r?\n---\r?\n'
 const dataSeparatorVertical = '\r?\n--\r?\n'
+const themeStore = useThemeStore()
+const isDarkMode = ref(themeStore.currentTheme.isDark)
```

Comment:
if that works well with watch
```suggestion
const { currentTheme } = useThemeStore()
const isDarkMode = ref(currentTheme.isDark)
```

Reply:
That's not working. The class `dark-mode` is not updating with the watch.